### ISSUE TITLE
Porcelain Melter always shows full lava

### ISSUE THREAD

**Issue by jaredlll08** [Darkosto/SkyFactory4Issues#90]
![image](https://user-images.githubusercontent.com/5553605/50542314-7f132480-0bc3-11e9-834a-6ecd31521330.png)
50mb of lava shows a full bar?

**Comment by MongoTheElder** [comment]
I always assumed it showed the lava needed to melt one item (50mB) and only filled if that much was available to be transferred from the tank below.

**Comment by Darkosto** [comment]
I believe it's meant to show the fuel used and the amount is in the tooltip. @KnightMiner could definitively answer this.

**Comment by KnightMiner** [comment]
Yeah, I noticed this recent in my personal pack, something with the tank checking code is getting confused. I think in the release version its not an issue, just the beta build I gave you (I really need to get around to making a release of that anyways)

**Comment by jaredlll08** [comment]
So the smelter now shows the actual lava value, but will display "No fuel found" if it hasn't smelted anything yet (place new smelter down, place lava, check gui)
https://cdn.discordapp.com/attachments/231741371847802880/544258172647505922/unknown.png
